hackvote/app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsController < ApplicationController
+	include ApplicationHelper
+
+	def create
+		@project = get_event.projects.build(project_params)
+		@project.project_id = @project.id
```

Comment:
explain to max/thomas what you are trying to do. seems like the way you are doing it is wrong.